mmcv/ops/nms.py

```diff
@@ -318,6 +318,7 @@ def batched_nms(boxes, scores, idxs, nms_cfg, class_agnostic=False):
 
     nms_type = nms_cfg_.pop('type', 'nms')
     nms_op = eval(nms_type)
+    score_dim = 5 if 'rotated' in nms_type else 4
```

Comment:
We need to add a comment. BTW, do we have a more elegant way to implement it?

Reply:
I'll add a comment about bbox shape.

I am considering the other implementation. But it is quite difficult because of TensorRT restriction. 
```
        # -1 indexing works abnormal in TensorRT
        # This assumes `dets` has 5 dimensions where
        # the last dimension is score.
        # TODO: more elegant way to handle the dimension issue.
        # Some type of nms would reweight the score, such as SoftNMS
```

Reply:
I think it's OK.